### PR TITLE
Login fix

### PR DESCRIPTION
**Pull request by rajikaimal**
Feature for checking whether an account is confirmed or not when a user login to the application.
## Description

When user login, if the account is not confirmed, it should display a message indicating that the account should be confirmed.

## Related Issue

#103 

## Motivation and Context

Feature addition. Improved logic.

## How Has This Been Tested?

Functional testing was carried out.
Environment : Ubuntu 14.04

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by rajikaimal** [comment]
Solves #103

**Comment by saisankargochhayat** [comment]
I think you should squash all your commits .

File: ui/src/app/completed/services/tableService.spec.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+describe('Service: TableService', function() {
```
Comment (malithsen) [inline ui/src/app/completed/services/tableService.spec.js]: These tests seem to fail

File: ui/src/app/completed/services/tableService.spec.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  beforeEach(inject(function (_TableService_) {
+    TableService = _TableService_;
+  }));
+
+  it('Should have TableService be defined', function () {
+    expect(TableService).toBeDefined();
+  });
+
+  describe('formatBytes', function() {
```
Comment (malithsen) [inline ui/src/app/completed/services/tableService.spec.js]: Don't we test `formatBytes` somewhere else?

File: ui/src/app/views/login.html [hunk 3]
```diff
@@ -17,6 +17,11 @@ <h1 class="md-display-2 login-heading">Login to Bassa</h1>
     transition: 0.3s;"class="closebtn" onclick="this.parentElement.style.display='none';">&times;</span>
   Incorrect Credentials
   </div>
+  <div style="padding:14px; background-color:grey; color: white;" class="alert" ng-if="unApproved">
```
Comment (malithsen) [inline ui/src/app/views/login.html]: How about we keep the styles in the stylesheets :)

**Comment by rajikaimal** [comment]
@malithsen I'll work on the changes, thank you for reviewing.

File: ui/src/app/views/login.html (hunk 3 above)
Comment (rajikaimal) [inline ui/src/app/views/login.html]: Good idea.

File: ui/src/app/completed/services/tableService.spec.js (hunk 1 above)
Comment (rajikaimal) [inline ui/src/app/completed/services/tableService.spec.js]: This is fixed with [9a50ccb](https://github.com/scorelab/Bassa/pull/104/commits/9a50ccbfbea3d69c5dbe0945a90126a18a1ff5c1)

**Comment by rajikaimal** [comment]
@malithsen Tests related to `table service` are not completed. I'll add a separate PR with some more tests.

File: ui/src/app/login/controllers/LoginCtrl.js [hunk 4]
```diff
@@ -9,13 +9,16 @@
 
     $scope.login = function(){
       $scope.incorrectCredentials = false;
+      $scope.unApproved = false;
       UserService.login($scope.user, function(status) {
-        if (status){
+        if (status.state == 200) {
```
Comment (saisankargochhayat) [inline ui/src/app/login/controllers/LoginCtrl.js]: Maybe use '===' operators ? they prevent unexpected type conversions :smile:

Comment (rajikaimal) [inline ui/src/app/login/controllers/LoginCtrl.js]: Yeah, but comes with a price too, it's an extra step for JavaScript to check the type so it affects performance.

File: ui/src/app/completed/services/tableService.spec.js (hunk 2 above)
Comment (rajikaimal) [inline ui/src/app/completed/services/tableService.spec.js]: Yes it was a mistake it's fixed now.

File: ui/src/app/views/login.html (hunk 3 above)
Comment (rajikaimal) [inline ui/src/app/views/login.html]: @malithsen Moved inline styles to style sheet with [0f31c25
](https://github.com/scorelab/Bassa/pull/104/commits/0f31c257077ec03ed0dc8af9e5d67df03a655639)

**Comment by rajikaimal** [comment]
@malithsen Can you review this :smiley:

**Comment by saisankargochhayat** [comment]
@malithsen Can we add this ?

**Comment by malithsen** [comment]
@saisankargochhayat @rajikaimal Hi guys sorry for taking this long. Let's fix the codacy suggestions and then do the merge

**Pull request merged**